src/pages/components/accordion/usage.mdx

```diff
@@ -8,7 +8,7 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 <PageDescription>
 
-An accordion is a vertically stacked list of headers that reveal or hide
+THE accordion is a vertically stacked list of headers that reveal or hide
```

Comment:
just wondering, where is this change coming from?

Reply:
ugh this idk what's going with my local environment - always pushes this change. removing now